pegasus/helpers/hoc_helpers.rb

```diff
@@ -1,5 +1,5 @@
 require 'rmagick'
-require 'cdo/graphics/certificate_image'
```

Comment:
some pegasus code was relying on this import in order to access `ScriptConstants`.